CHANGELOG.md

```diff
@@ -79,6 +80,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
 
 <!-- next-url -->
+
```

Comment:
The line doesn't need to be there, it might bother the automatic generation of changelog for releases

Reply:
oops, had format-on-save on and prettier has some strong opinions on markdown formatting, removed